schema/meta_schema_language_cpp.yaml

```diff
@@ -56,6 +56,10 @@ typeSupportStatuses:
     status: supported
     notes: ""
     propertyOverrides: []
+  - type: ConsoleMetricExporter
```

Comment:
@marcalff I merged some other PRs that created conflicts with this. I went ahead and resolved, but have made an assumption here c++ supports (or will imminently) support the new console metric exporter type.

Let me know and I'll get this merged.

Reply:
Yes @jack-berg , C++ will do that very soon.

https://github.com/open-telemetry/opentelemetry-cpp/blob/930b694af4f4c96f98d85514313d1dbaa8ed4b06/sdk/src/configuration/configuration_parser.cc#L608

Ok to merge.